l10n_ar_stock/README.rst

```diff
@@ -16,14 +16,12 @@ Integración del módulo de stock a requerimientos argentinos
 
 #. Remito electrónico Argentino
 #. Número de despacho en lotes
-#. Crea libro de stock en caso de que no exista
+#. Crea talonario de remito en caso de que no exista
 
 Sobre remito electrónico
 ------------------------
 
 PENDIENTE:
-
```

Comment:
re add space between pendientes y and the list, if you remove the space the result is that the view is wrongly shown (all the pendings in one line(

![image](https://github.com/user-attachments/assets/9ce0d253-3db4-4ebf-90a4-d8e1aac1e9af)

If you leave the empty line will work as before (see bellow(. onyl remove the first item if you needed 
![image](https://github.com/user-attachments/assets/f53345b5-050e-49fc-9668-c5c70d9e6a1f)